src/Client.js

```diff
@@ -259,6 +260,24 @@ Client.prototype.syncLastTxnTime = function(time) {
   this._http.syncLastTxnTime(time)
 }
 
+/**
+ * Closes the client session and cleans up any held resources.
+ * By default, it will wait for any ongoing requests to complete on their own;
+ * streaming requests are terminated forcibly. Any subsequent requests will
+ * error after the .close method is called.
+ * Should be used at application termination in order to release any open resources
+ * and allow the process to terminate e.g. when the custom http2SessionIdleTime parameter is used.
+ *
+ * @param {?object} opts Close options.
+ * @param {?boolean} opts.force Specifying this property will force any ongoing
+ * requests to terminate instead of gracefully waiting until they complete.
```

Comment:
Currently not true for the implementation of HTTP1. Either the docs or implementation should change. My preference would be for implementation to match this description -- it's easier for users to understand if the behavior is consistent (HTTP1 vs. HTTP2 is an implementation detail that most users won't care much about).